src/components/grid/grid.js

```diff
@@ -44,12 +44,11 @@ const Column = props => (
     {...props}
     // Padding right/left. 0-4 represent index on the spacing scale. Numbers 5+ represents px value
     px={[
-      1, // Padding between 0px and 1st breakpoint (375px). 1 = 5px on spacing scale
-      2, // Padding between 1st breakpoint(375px) and 2nd breakpoint (768px). 2 = 10px on spacing scale
-      2, // Padding between 2nd breakpoint(768px) and 3rd breakpoint (1280px). 2 = 10px on spacing scale
-      3 // Padding between 3rd breakpoint(1280px) and 4th breakpoint (1440px). 3 = 15px on spacing scale
+      1, // pad btwn 0px and 1st breakpoint (375px). 1 = 5px on spacing scale
```

Comment:
FYI i shortened this because the comment went beyond the character per line limit set by our linter